helpers/GenerateResourcesAndImage.ps1

```diff
@@ -192,6 +192,10 @@ Function GenerateResourcesAndImage {
     # Resource group names may contain special characters, that are not allowed in the storage account name
     $storageAccountName = $storageAccountName.Replace("-", "").Replace("_", "").Replace("(", "").Replace(")", "").ToLower()
     $storageAccountName += "001"
+    
+    
+    # Storage Account Name can only be 24 characters long
+    $storageAccountName = $storageAccountName.Substring(0, 24)
```

Comment:
You should check if the length of storageAccountName is more than 24:
```
PS > "ahbfff".Substring(0, 24)
MethodInvocationException: Exception calling "Substring" with "2" argument(s): "Index and length must refer to a location within the string. (Parameter 'length')"
```

Reply:
Somehow missed this completely. Added a check for the length